### DISCUSSION TITLE
GSSoC'24 Leaderboard update

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/Stackoverflow-Analysis#258] [Announcements]
## Leaderboard Achievements 🏆

![gssoc_leaderboard (1024 x 728 px) (1024 x 728 px)](https://github.com/Recode-Hive/Stackoverflow-Analysis/assets/30715153/942e723e-0dfd-4bf2-9edb-593f23d42015)

#### To see what it looks like visit your Profile on  [Leaderboard](https://gssoc.girlscript.tech/leaderboard) <br>
Note: [Leaderboard](https://gssoc.girlscript.tech/leaderboard) available from 30th May  :octocat: 

Why Achievement badge: The reason you are here is to do open-source, and it's not just about doing code, it's about helping the community, So rather than getting a score we want to give you acknowledgment of your contribution from all different levels whether you a beginner or Expert and different perks when you achieve the same.

Other people who visit your profile can see how helpful you are in open source by analyzing the badge. So are you ready?

#### You can achieve all the badges when you attain the desired scores, Check the Score list below 👇🏻

# 📃 Achievement List 📃
<br>

| Badge | Name | How to get | Needed Score | 
| :-: | :-: | :-: | :-: |
| ![]
![image](https://github.com/Recode-Hive/Stackoverflow-Analysis/assets/30715153/f3045ade-446f-46d8-a730-05f9f2ca9ecf)                 | Explorer Badge       |   Unlock the Explorer Badge for reaching 60 points. Keep exploring and discovering new horizons | <table>  <thead> 60 Points</tr>   </tbody>  </table>      |
![image](https://github.com/Recode-Hive/Stackoverflow-Analysis/assets/30715153/9666dc59-e8d8-4273-b110-ce4f26e2f3b2)| Adventurer Badge       |   You're now an Adventurer! With 140 points, you've proven your knack for navigating through challenges. K | <table>  <thead> 140 Points</tr>   </tbody>  </table>      |
![image](https://github.com/Recode-Hive/Stackoverflow-Analysis/assets/30715153/74e579a0-a2b4-4010-9c8c-271a51cc441d)                 | Trailblazer Badge       |   Blaze your trail! Unlock the Trailblazer Badge by an amazing 200 points. Your determination and courage inspire us all! | <table>  <thead> 200 Points</tr>   </tbody>  </table>      |
![image](https://github.com/Recode-Hive/Stackoverflow-Analysis/assets/30715153/93a7cce8-595b-4c65-83fb-c3e92241f0ff)            | Summit Seeker Badge       |   You've conquered mountains! Unlock with 300 points, you've earned the Summit Seeker Badge. | <table>  <thead> 300 Points</tr>   </tbody>  </table>      |
![image](https://github.com/Recode-Hive/Stackoverflow-Analysis/assets/30715153/2274e5cd-fb05-43fc-a939-3e18ca881c1b)          | Champion Badge    |   A true champion! Unlock with 500 points, you've reached the pinnacle of success and earned the Champion Badge.  | <table>  <thead> 500 Points</tr>   </tbody>  </table>      |
![image](https://github.com/Recode-Hive/Stackoverflow-Analysis/assets/30715153/3ca70186-0e49-442b-85b2-061d819cfd8e)     | Innovator Badge    |   Innovate Further! Unlock with 1200 points, you've reached the one of the milestone. Keep Climbing to new heights  | <table>  <thead> 1200 Points</tr>   </tbody>  </table>      |
![image](https://github.com/Recode-Hive/Stackoverflow-Analysis/assets/30715153/8a0a8b82-c425-4627-ba02-75beb47198d6)    | Conqurer Badge    |   Master Level Claim! Unlock with 2500 points. Your determination and courage inspire us all!  | <table>  <thead> 2500 Points</tr>   </tbody>  </table>      |
![image](https://github.com/Recode-Hive/Stackoverflow-Analysis/assets/30715153/0ead0a40-e979-41e2-aef9-7798f044c79f)    | Legend Badge    |   "You're now an Legend! Unlock with 5500 points, Keep journeying towards leaderboard Champion" | <table>  <thead> 5500 Points</tr>   </tbody>  </table>      |

More badges will roll on June 22nd with Perks...

Things to Note: 👇🏻
- [X]  Labels should be assigned with your name and levels on PR (Pull Request)
- [X]  Labels such as gssoc and any of level1, level2, level3 should be there.
- [X] At a time only a level can given to a PR. Eg: level 1. 
- [X] Even if you contributed before and no labels were assigned you can reach out to the Project Admin and get labels.
- [X] Leaderboard updates twice in a day, so be patient. Changes might not appear instant.

**Comment by piyushsahujtp** [comment]
I am unable to find my name on the leaderboard , why ?

**Reply by sanjay-kv** [reply]
give me any PR you merged. I can have a look into it.
